protocol/protocol.proto

```diff
@@ -22,556 +22,556 @@ message Miniblock {
-    }
+  message Inception {
+    bytes stream_id = 1;
+    EncryptedData channel_properties = 2;
+    StreamSettings settings = 3;
+  }
 
-    oneof content {
-        Inception inception = 1;
-        EncryptedData message = 2;
-        EncryptedData channel_properties = 3;
-    }
+  oneof content {
+    Inception inception = 1;
+    EncryptedData message = 2;
+    EncryptedData channel_properties = 3;
+  }
 }
 
 /**
 * UserPayload
 */
 message UserPayload {
-    message Snapshot {
-        // inception
-        Inception inception = 1;
-        // memberships, sorted by stream_id
-        repeated UserMembership memberships = 2;
-    }
+  message Snapshot {
+    // inception
+    Inception inception = 1;
+    // memberships, sorted by stream_id
+    repeated UserMembership memberships = 2;
+  }
+
+  message Inception {
+    bytes stream_id = 1;
+    StreamSettings settings = 2;
+  }
 
-    message Inception {
-        bytes stream_id = 1;
-        StreamSettings settings = 2;
-    }
-    
-    // update own membership
-    message UserMembership {
-        bytes stream_id = 1;
-        MembershipOp op = 2;
-        optional bytes inviter = 3;
-        optional bytes stream_parent_id = 4;
-    }
+  // update own membership
+  message UserMembership {
+    bytes stream_id = 1;
+    MembershipOp op = 2;
+    optional bytes inviter = 3;
+    optional bytes stream_parent_id = 4;
+  }
 
-    // update someone else's membership
-    message UserMembershipAction {
-        bytes stream_id = 1;
-        bytes user_id = 2; 
-        MembershipOp op = 3;
-        optional bytes stream_parent_id = 4;
-    }
+  // update someone else's membership
+  message UserMembershipAction {
+    bytes stream_id = 1;
+    bytes user_id = 2;
+    MembershipOp op = 3;
+    optional bytes stream_parent_id = 4;
+  }
 
-    oneof content {
-        Inception inception = 1;
-        UserMembership user_membership = 2;
-        UserMembershipAction user_membership_action = 3;
-    }
+  oneof content {
+    Inception inception = 1;
+    UserMembership user_membership = 2;
+    UserMembershipAction user_membership_action = 3;
+  }
 }
 
 /**
 * UserInboxPayload
 * messages to a user encrypted per deviceId
 */
 message UserInboxPayload {
-    message Snapshot {
-        message DeviceSummary {
-            /**
-            * UpperBound = latest to device event sent from other client per deviceKey
-            * LowerBound = latest ack sent by stream owner per deviceKey
-            * on ack, if UpperBound <= LowerBound then delete this deviceKey entry from the record
-            * on ack or new session, if any device’s lower bound < N generations ago, delete the deviceKey entry from the record
-            */
-            int64 lower_bound = 1;
-            int64 upper_bound = 2;
-        }
-
-        Inception inception = 1;
-        // deviceKey: miniblockNum that the ack was snapshotted
-        map<string, DeviceSummary> device_summary = 2;
+  message Snapshot {
```

Comment:
can you make this protocol.proto formatting change in a different commit? Is this hooked up to a linter somewhere? Pretty sure vscode is just going to undo this.